modules/contact/include/userobjects/PenaltyFrictionUserObject.h

```diff
@@ -52,33 +56,19 @@ class PenaltyFrictionUserObject : public WeightedVelocitiesUserObject
   /// The friction coefficient
   const Real _friction_coefficient;
 
-  /// Map from degree of freedom to old weighted gap
-  std::unordered_map<const DofObject *, ADReal> _dof_to_old_weighted_gap;
-
-  /// Map from degree of freedom to old physical (not weighted) tangential velocity
-  std::unordered_map<const DofObject *, std::array<ADReal, 2>> _dof_to_old_real_tangential_velocity;
-
-  /// Map from degree of freedom to accumulated slip
-  std::unordered_map<const DofObject *, std::array<ADReal, 2>> _dof_to_accumulated_slip;
-
-  /// Map from degree of freedom to old accumulated slip
-  std::unordered_map<const DofObject *, std::array<ADReal, 2>> _dof_to_old_accumulated_slip;
+  /// Map from degree of freedom to current and old accumulated slip
```

Comment:
It'd be nice to indicate in the comment that slip is only used for output purposes.